### PR TITLE
#fix: minor fixes

### PR DESCRIPTION
**Pull request by MayankKumar10**

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
looks good to me

**Review by anik31 — APPROVED** [review]
LGTM

File: src/styles/Footer.css [hunk 1]
```diff
@@ -1,13 +1,13 @@
 .FooterContainerMain {
   display: flex;
-  flex-direction: row;
-  align-items: center;
-  flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: space-evenly;
   background-color: rgb(48, 43, 43);
```
Comment (vrinda-mahajan) [inline src/styles/Footer.css:4]: Create a variable for the color.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

**Review by vrinda-mahajan — APPROVED** [review]
Everything else looks good.

File: src/styles/Footer.css (hunk 1 above)
Comment (MayankKumar10) [reply src/styles/Footer.css:4]: > Create a variable for the color.

Thank you for the suggestion, I will

**Comment by MayankKumar10** [comment]
> Everything else looks good.

Thank you

**Comment by MayankKumar10** [comment]
> looks good to me

Thank you Arjun, can you suggest to me anything to improve it

**Pull request closed without merging**